tests/e2e/utils/file-reset.ts

```diff
@@ -50,6 +50,6 @@ export const resetFiles = async () => {
   // wait for server to restart after modifying files
   console.log('Waiting for Strapi to restart...');
   // TODO: this is both a waste of time and flaky. We need to find a way to access playwright server output and watch for the "up" log to appear
-  await delay(3); // give it time to detect file changes and begin its restart
+  await delay(4); // give it time to detect file changes and begin its restart
```

Comment:
Seems arbitrary; do we have any idea if this could cause problems (flaky tests) in the future? Is this impacted by servers' performances?

Reply:
It is arbitrary. It is based on the server performance, and the CI is currently much much slower than running tests locally, so this just adds 1s to every fileReset to give Strapi server more time to realize that files have changed and resart itself.

Maybe a future improvement could just be to shut down Strapi, reset the files, then start up Strapi again, but I don't know if that would help.

Essentially, I found that the primary reason that the CTB tests were failing was just timing out, with a secondary cause being that in some cases (I couldn't determine when) having a fileReset before a resetDatabase was causing tests to fail.